example/lib/pages/movie_image_page.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:example/utils/string_constants.dart';
+import 'package:flutter/material.dart';
+import 'package:movie_widgets/movie_widgets.dart';
+
+class MovieImagePage extends StatelessWidget {
+  const MovieImagePage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        backgroundColor: Colors.grey,
+        leading: IconButton(
+          onPressed: () => Navigator.of(context).pop(),
+          icon: Icon(
+            Icons.arrow_back,
+            color: Colors.black,
+          ),
+        ),
+      ),
+      body: MovieImage(
+        posterPath: StringConstants.posterPath,
+      ),
+    );
```

Comment:
You're using the same code to create all the pages but the only change that you do is the body. So you can create only a  widget of Page (line 10 to 20 is the same) but you need to add a parameter that is going to be a Widget

Reply:
done